src/sentry/api/serializers/rest_framework/group_notes.py

```diff
@@ -3,21 +3,62 @@
 from rest_framework import serializers
 
 from .list import ListField
+from sentry.api.fields.actor import ActorField
+
+from sentry.models import User, Team
+
+
+def seperate_actors(actors):
+    users = [actor for actor in actors if actor.type is User]
+    teams = [actor for actor in actors if actor.type is Team]
+
+    return {'users': users, 'teams': teams}
+
+
+def seperate_resolved_actors(actors):
+    users = [actor for actor in actors if isinstance(actor, User)]
+    teams = [actor for actor in actors if isinstance(actor, Team)]
+
+    return {'users': users, 'teams': teams}
```

Comment:
i think some comments about what these two functions do/how they are different could be helpful